produce/lib/produce/developer_center.rb

```diff
@@ -18,8 +18,42 @@ def create_new_app
         app_name = Produce.config[:app_name]
         UI.message "Creating new app '#{app_name}' on the Apple Dev Center"
 
+        allowed_keys = [:app_group, :apple_pay, :associated_domains, :data_protection, :game_center, :health_kit, :home_kit,
+                        :wireless_accessory, :icloud, :in_app_purchase, :inter_app_audio, :passbook, :push_notification, :siri_kit, :vpn_configuration]
+
+        Produce.config[:enabled_features].select { |key, value| allowed_keys.include? key }
```

Comment:
I think this is an input validation concern, and should be handled either in a validation block for the option, or in a `detect_values` file

Also, does this line of code do what we want? I don't think `select` will modify a Hash in-place, so I don't think the work done here is picked up on line 27?